README.md

```diff
@@ -62,24 +62,67 @@ npm install @google-cloud/dialogflow-cx
  * TODO(developer): Uncomment these variables before running the sample.
  */
 // const projectId = 'my-project';
-// const location = 'us';
-// const agent = 'foo';
+// const location = 'global';
+// const agentId = 'my-agent';
+// const audioFileName = '/path/to/audio.raw';
+// const encoding = 'AUDIO_ENCODING_LINEAR_16';
+// const sampleRateHertz = 16000;
+// const languageCode = 'en'
 
 // Imports the Google Cloud Some API library
-const {IntentsClient} = require('@google-cloud/dialogflow-cx');
-const client = new IntentsClient();
-async function listIntents() {
-  const parent = client.agentPath(projectId, location, agent);
-  console.info(parent);
-  // TODO: implement a quickstart that does something useful:
-  /*
-  const [intents] = await client.listIntents({
-    parent,
-  });
-  console.info(intents);
-  */
+const {SessionsClient} = require('@google-cloud/dialogflow-cx');
+const client = new SessionsClient();
+
+const fs = require('fs');
+const util = require('util');
+
+async function detectIntentAudio() {
+  console.info('gots here');
+  const sessionId = Math.random().toString(36).substring(7);
```

Comment:
Could we use the `uuid` library here similar to the Dialogflow quickstart: (https://www.npmjs.com/package/dialogflow)? 

```js
const uuid = require('uuid');
sessionId = uuid.v4();
```